addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -438,7 +438,6 @@ $o-wblog-title-sizes-variants: (
         .s_latest_posts_post_title  {
             @include font-size($h3-font-size);
             margin-bottom: 0.5em;
-            word-spacing: -0.15em;
```

Comment:
SRI and his negative word spacing :grimacing: :heart: 

Definitely something to remove in stable if not already (probably done already?) but I would not do in stable IMO. What do you think @rdeodoo ?

Reply:
> SRI and his negative word spacing  😬 ❤️
😂  

 @bso-odoo , it's definitively to remove. It was initially introduced to make the default design nicer, but it obviously conflicts with the overall architecture, especially because the user doesn't have control over these values.

Reply:
> What do you think @rdeodoo ?

I think a one line scss PR should at least have a before/after image to easily get what it does. At least in the PR description

Reply:
I guess it does not really qualifies as a bug and it doesn't fit the stable policy but.. It looks really broken without the fix, which is rather harmless right?
I guess that's not that critical on the other hand since it wasn't reported before.
99% of the time either 13.0 or master. Here I'd say 15.0 or master, considering the risk/reward pretty good on our latest stable.